src/WebJobs.Script/Description/Node/Script/globalInitialization.js

```diff
@@ -8,6 +8,9 @@ return function (context, callback) {{
         context.handleUncaughtException(err.stack);
     }});
 
+    // workaround for https://github.com/tjanczuk/edge/issues/325
+    process.nextTick = global.setImmediate;
```

Comment:
Please put a TEMP HACK comment on this so we remove it. Also recommend logging a new issue in our repo to clean this up.